docs/getting-started/intro-to-dockstore-tools-and-workflows.rst

```diff
@@ -1,27 +1,21 @@
-Introduction to Workflows, Tools, and Services
-==============================================
+Introduction to Workflows and Tools
+====================================
+
+Tools and workflows hosted on Dockstore are computational programs described in languages that make it easy for their reuse in different compute environments. Generally, descriptor languages leverage a containerized compute environment (such as a Docker image) that packages up all of the code and dependencies needed to complete the computational step.
+
+Not all descriptor languages make a distinction between a tool and a workflow. Most users will register workflows on Dockstore. 
 
 What is a workflow?
 -------------------
 
-In general terms, a workflow is a chain of commands that has several steps linked together, which create some sort of final output. For example, a simple two-step workflow that converts a VCF into a GDS file may conceptually look like this:
+In general terms, a workflow describes a chain of commands that may be executed in a specified order. For example, a simple two-step workflow that converts a VCF into a GDS file may conceptually look like this:
 
-vcf file --> convert to GDS file --> give each variant a unique ID --> a GDS file with unique IDs
+VCF file --> convert to GDS file --> give each variant a unique ID --> a GDS file with unique IDs
 
 In this case, the output of the conversion step is the input of the step that generates unique IDs, and the unique IDs step's output is the final output of the workflow: A GDS file with unique IDs. 
 
-Managing workflows can be complicated, as you may be chaining together many steps, each with different computational requirements. This is where workflow languages step in, allowing us to create formal "tasks" which can each have their own computational settings or Docker container, and chaining the output of one task into the input of another task.
+Managing workflows can be complex, as you may be chaining together many steps, each with different computational requirements. This is where workflow languages step in, allowing us to create formal "tasks" which can each have their own computational settings or Docker container, and chaining the output of one task into the input of another task.
 
 What is a tool?
 ---------------
-How Dockstore defines a tool is changing, but our aim is to align our tool definition with that of the languages we support.
-For the sake of simplicity, Dockstore only recommends registering a Dockstore tool when your work is a CWL CommandLineTool.
-
-If you want a deeper understanding of the history of Dockstore tools and where we plan on going, read :doc:`Dockstore Tools Overhaul </../advanced-topics/dockstore-tools-overhaul>`.
```

Comment:
Think the link makes sense and already had the right level of suggestion

Reply:
@denis-yuen, do you want me to put this text back?

Reply:
yeah, I think so

Reply:
@denis-yuen In my opinion, this page is aimed directly at new users that don't know what a tool or workflow is. I don't think they need this extra information and it could be more confusing than helpful to them. 

This link is only relevant to returning users. Returning users should be about to find the tool overhaul and how it affects them in our tool registration and advanced topics pages.

Reply:
There's an "if you want", I think they can judge for themselves if they want a deeper understanding. 
It looks like the page could be useful for helping users decide between our various registration methods.